NitroxLauncher/LauncherLogic.cs

```diff
@@ -258,7 +258,7 @@ private Process StartSubnautica()
 
             if (PlatformDetection.IsEpic(subnauticaPath))
             {
-                startInfo.Arguments = "-EpicPortal";
+                startInfo.Arguments = "-EpicPortal -vrmode none";
             }
```

Comment:
Do we need to have this argument for all platforms?  Should we make it a setting in the launcher?

Reply:
Its not an issue with Steam, could be an issue with other platforms however but don't quote me